apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000019_reduce_trace_last_updated_at_to_micros.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+--changeset andrescrz:000019_reduce_trace_last_updated_at_to_micros
+
+ALTER TABLE ${ANALYTICS_DB_DATABASE_NAME}.traces
+    MODIFY COLUMN last_updated_at DateTime64(6, 'UTC') DEFAULT now64(6);
```

Comment:
As per documentation: https://docs.liquibase.com/workflows/liquibase-community/automatic-custom-rollbacks.html

```suggestion
    MODIFY COLUMN last_updated_at DateTime64(6, 'UTC') DEFAULT now64(6);
    
 --rollback empty
```

Reply:
Perfect. I didn't know about this and we didn't have any previous example. We should use this from now onward.